server/channels/app/export_test.go

```diff
@@ -240,17 +244,18 @@ func TestExportDMChannel(t *testing.T) {
 		// DM Channel
 		ch := th1.CreateDmChannel(th1.BasicUser2)
 
-		th1.App.Srv().Store().Preference().Save(model.Preferences{
+		err := th1.App.Srv().Store().Preference().Save(model.Preferences{
 			{
 				UserId:   th1.BasicUser2.Id,
 				Category: model.PreferenceCategoryFavoriteChannel,
 				Name:     ch.Id,
 				Value:    "true",
 			},
 		})
+		require.NoError(t, err)
 
 		var b bytes.Buffer
-		err := th1.App.BulkExport(th1.Context, &b, "somePath", nil, model.BulkExportOpts{})
+		err = th1.App.BulkExport(th1.Context, &b, "somePath", nil, model.BulkExportOpts{})
```

Comment:
`err` is now an `error`, but `th1.App.BulkExport` returns a `*model.AppError`. Please use a new variable.